content/automate/ManageIQ/Transformation/TransformationHosts/Common.class/__methods__/vmtransform.rb

```diff
@@ -31,31 +25,23 @@ def main
 
               @handle.log(:info, "Transformation - Started On: #{start_timestamp}")
 
-              destination_ems = @handle.vmdb(:ext_management_system).find_by(:id => task.get_option(:destination_ems_id))
-              max_runners = Transformation::TransformationHosts::Common::Utils.ems_max_runners(destination_ems, factory_config)
-              if Transformation::TransformationHosts::Common::Utils.get_runners_count_by_ems(destination_ems, factory_config) >= max_runners
-                @handle.log(:info, "Too many transformations running [#{max_runners}]. Retrying.")
-              else
-                wrapper_options = ManageIQ::Automate::Transformation::TransformationHosts::Common::Utils.virtv2vwrapper_options(task)
+              wrapper_options = ManageIQ::Automate::Transformation::TransformationHosts::Common::Utils.virtv2vwrapper_options(task)
 
-                # WARNING: Enable at your own risk, as it may lead to sensitive data leak
-                # @handle.log(:info, "JSON Input:\n#{JSON.pretty_generate(wrapper_options)}") if @debug
+              # WARNING: Enable at your own risk, as it may lead to sensitive data leak
+              # @handle.log(:info, "JSON Input:\n#{JSON.pretty_generate(wrapper_options)}") if @debug
 
-                @handle.log(:info, "Connecting to #{transformation_host.name} as #{transformation_host.authentication_userid}") if @debug
-                @handle.log(:info, "Executing '/usr/bin/virt-v2v-wrapper.py'")
-                result = Transformation::TransformationHosts::Common::Utils.remote_command(task, transformation_host, "/usr/bin/virt-v2v-wrapper.py", wrapper_options.to_json)
-                raise result[:stderr] unless result[:rc].zero?
+              @handle.log(:info, "Connecting to #{transformation_host.name} as #{transformation_host.authentication_userid}") if @debug
+              @handle.log(:info, "Executing '/usr/bin/virt-v2v-wrapper.py'")
```

Comment:
@fdupont-redhat 
Do we have roles enabled for these tasks to be reliably run on a server where the code exists?

Reply:
Yes, the role is enabled as we filter out the hosts in AcquireTransformationHost.